db/index.js

```diff
@@ -0,0 +1,434 @@
+    queryProductString = '';
+    countProduct = 0;
+  }
+};
+productRl.on('close', () => {
+  const categoryString = ` ('${categories.join("'), ('")}')`;
+
+  pool.query(
+    `INSERT INTO categories(category_name) VALUES ${categoryString}`,
+    (err, result) => {
+      if (err) {
+        console.error('Error executing query', err.stack);
+      }
+
+      for (let i = 0; i < queryProductArray.length; i += 1) {
+        const productString = queryProductArray[i].substring(0, queryProductArray[i].length - 2);
+        pool.query(
+          `INSERT INTO products(id, name, slogan, description, category_id, default_price) VALUES ${productString}`,
+          (productErr, productResult) => {
+            if (productErr) {
+              console.error('Error executing query', productErr.stack);
+            }
+            return productResult;
+          }
+        );
+      }
+      const extraProductQuery = queryProductString.substring(0, queryProductString.length - 2);
+      pool.query(
+        `INSERT INTO products(id, name, slogan, description, category_id, default_price) VALUES ${extraProductQuery}`,
+        (err, result) => {
+          if (err) {
+            console.error('Error executing extra query', err.stack);
+          }
+          featurePort();
+          return result;
+        }
+      );
+    }
+  );
+});
+
+// Related CSV Handler
+let relatedFilename = "./server/related.csv";
+
+// Use fs.createReadStream() method
+// to read the file
+
+let relatedReader = fs.createReadStream(relatedFilename);
+let relatedRl = readline.createInterface({
+  input: relatedReader,
+  crlfdelay: Infinity,
+});
+relatedRl.on("line", function (line) {
+  handleRelatedLine(line);
+});
+let queryRelatedString = "";
+let countRelated = 0;
+
+let queryRelatedArray = [];
+let handleRelatedLine = (line) => {
+  line = line.split(/,(?=(?:(?:[^"]*"){2})*[^"]*$)/);
+  for (let j = 0; j < line.length; j += 1) {
+    if (line[j][0] === '"') {
+      line[j] = line[j].substr(1);
+    }
+    if (line[j][line[j].length - 1] === '"') {
+      line[j] = line[j].substr(0, line[j].length - 1);
+    }
+  }
+  queryRelatedString += `('${line[1]}', '${line[2]}'), `;
+  countRelated += 1;
+  if (countRelated === 1000) {
+    // querySkusArray.push(querySkusString);
+    queryRelatedString = queryRelatedString.substring(0, queryRelatedString.length - 2);
+    // console.log('skusString:', querySkusString);
+    pool.query(
+      `INSERT INTO related(current_product_id, related_product_id) VALUES ${queryRelatedString}`,
+      (nextErr, nextResult) => {
+        if (nextErr) {
+          console.error("Error executing query", nextErr.stack);
+        }
+      }
+    );
+    queryRelatedString = "";
+    countRelated = 0;
+  }
+};
+relatedRl.on('close', () => {
+  const extraRelatedQuery = queryRelatedString.substring(0, queryRelatedString.length - 2);
+  pool.query(
+    `INSERT INTO related(current_product_id, related_product_id) VALUES ${extraRelatedQuery}`,
```

Comment:
Here you could consider passing in the values as an array. I think the `pg` library allows you to pass two arguments into the pool.query method 
1) query string (basically what you have just w/ placeholders ($1, $2, $3...)) 
2) query args (an array of values to be inserted into the query string)